coordinator/persistence-api/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Copied from persistence backends' test setups, to reduce noise from Netty DEBUG outpit.